test/java/build.xml

```diff
@@ -20,6 +20,7 @@
   <property name="samples.src.dir" location="../../samples/java/functional/src"/>
   <property name="testtk" location="../spl/testtk"/>
   <property name="testtk.java" location="${testtk}/impl/java"/>
+  <property name="testtkopt" location="../spl/testtkopt"/>
```

Comment:
I think this will break the build when the version required by the toolkit is not available???